src/index.ts

```diff
@@ -4,4 +4,4 @@ const semanticReleaseMajorTag = {
   success,
 };
 
-export default semanticReleaseMajorTag;
+export = semanticReleaseMajorTag;
```

Comment:
Thank you, that's a perfect spot, not sure how I didn't notice this.. That's what happens when you don't test properly haha...
```
export { success };
```
could also be an option, but if this works fine for you then it should be totally fine also :smiley:
Thanks 💪